xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -123,7 +128,17 @@ export default {
       const noOtherEditing = this.logic.getEditBus().isEditable()
       return editable && noOtherEditing;
     },
-    
+    isViewable() {
+      var empty = this.isEmpty;
+      if (empty === undefined) {
+        empty = !this.value
+      }
+      const isViewable = this.logic.isActionAllowed('view', this.entry) || !empty;
+      if (!isViewable) {
+        this.logic.footnotes.put(this.$t('livedata.displayer.emptyValue'), 'livedata.footnotes.propertyNotViewable');
```

Comment:
The fact that this function has side effects (adds the footnote) is a but unexpected (for a computed property that returns boolean). I don't have a better idea though.